packages/react-composites/src/composites/common/icons.tsx

```diff
@@ -11,15 +11,25 @@ import {
 } from '@fluentui/react-icons';
 import { ComponentIcons, DEFAULT_COMPONENT_ICONS } from '@internal/react-components';
 import React from 'react';
+import { mergeStyles, Text } from '@fluentui/react';
+
+const CoffeeIcon = (): JSX.Element => <Text className={mergeStyles(coffeeIconStyle)}>☕</Text>;
```

Comment:
An aria-doNotReadThis thing needing added for screen reader?

Reply:
aria-hidden={true}